contentcuration/contentcuration/frontend/channelEdit/components/AssessmentItemEditor/AssessmentItemEditor.spec.js

```diff
@@ -9,7 +10,13 @@ const store = factory();
 jest.mock('shared/views/MarkdownEditor/MarkdownEditor/MarkdownEditor.vue');
 jest.mock('shared/views/MarkdownEditor/MarkdownViewer/MarkdownViewer.vue');
 
+const listeners = {
```

Comment:
I can't see `listeners` technique to be used in `AssessmentItemEditor`. Is there a reason to update tests in this file to use `listeners` instead of `wrapper.emitted()`?